klippy/extras/ads1100.py

```diff
@@ -64,10 +64,10 @@ def get_last_value(self):
 
 
     def read_single_value(self):
-        
+
```

Comment:
Please leave out these space-only changes. It will clutter the merge commit for the config file. It will make things more complicated for me when I want to create pull requests to upstream.

(If this were something more complicated I would ask for a separate merge request, but in this case I will simply fix these mistakes myself soon...)